packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx

```diff
@@ -165,6 +167,20 @@ export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
       setDrawerMenuItems([]);
     }
   }, [participantActioned, remoteParticipants]);
+
+  /* @conditional-compile-remove(teams-meeting-conference) */
+  const conferencePhoneInfo = adapter.getState().call?.teamsMeetingConference;
```

Comment:
use a selector with useSelector for this data. This ensures if it ever changes this component rerenders with the latest data